docs/sources/collect/datadog-traces-metrics.md

```diff
@@ -122,18 +122,21 @@ The [otelcol.exporter.otlp][] component is responsible for delivering OTLP data
     - _`<USERNAME>`_: The basic authentication username.
     - _`<PASSWORD>`_: The basic authentication password or API key.
 
-## Configure Datadog Agent to forward metrics to the Datadog Receiver
+## Configure Datadog Agent to forward telemetry to the {{< param "PRODUCT_NAME" >}} Datadog Receiver
 
-You can set up your Datadog Agent to forward Datadog metrics simultaneously to {{< param "PRODUCT_NAME" >}} and Datadog.
+You can set up your Datadog Agent to forward Datadog metrics and traces simultaneously to {{< param "PRODUCT_NAME" >}} and Datadog.
 
 We recommend this approach for current Datadog users who want to try using {{< param "PRODUCT_NAME" >}}.
 
-1. Add the following environment variable to your datadog-agent installation.
+1. Add the following environment variables to your Datadog-agent installation.
 
    ```bash
    DD_ADDITIONAL_ENDPOINTS='{"http://<DATADOG_RECEIVER_HOST>:<DATADOG_RECEIVER_PORT>": ["datadog-receiver"]}'
+   DD_APM_ADDITIONAL_ENDPOINTS='{"http://<DATADOG_RECEIVER_HOST>:<DATADOG_RECEIVER_PORT>": ["datadog-receiver"]}'
    ```
 
+   `DD_ADDITIONAL_ENDPOINTS` is used for forwarding metrics, whereas `DD_APM_ADDITIONAL_ENDPOINTS` is for traces.
```

Comment:
@alfredo-d I think this is what you had in mind?

Reply:
heya @ptodev , yeah that looks about right. Thanks for this!